docs/source/algorithms/ConjoinWorkspaces-v1.rst

```diff
@@ -66,12 +66,15 @@ y axis unit and label when conjoining workspaces. Changing YAxisUnit updates
 YAxisLabel automatically with the value of YAxisUnit, unless a separate value
 is supplied.
 
-Exception
-#########
+Exceptions
+##########
+
+If property 'CheckMatchingBins' is set to true, and the bins in the two input workspaces
+do not match, then an 'invalid_argument' exception will be thrown from the 'exec' method.
```

Comment:
Having some monospace formatting here and on the lines below would improve readability
```suggestion
If property ``CheckMatchingBins`` is set to true, and the bins in the two input workspaces
do not match, then an ``invalid_argument`` exception will be thrown from the ``exec`` method.
```